test/test_utils/sagemaker.py

```diff
@@ -3,7 +3,6 @@
 import subprocess
 import random
 import re
-
```

Comment:
nit: i believe this newline should not be removed

Reply:
I specifically removed it here to make it follow the import convention on Python.

Reference:
https://www.python.org/dev/peps/pep-0008/
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement